### PR TITLE
Fix javadoc for browse sdk

### PR DESCRIPTION
**Pull request by varungu**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @varungu has signed the CLA. Thanks for the pull request!

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseActivity.java [hunk 1]
```diff
@@ -150,12 +160,22 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+     * @return the top browse fragment
+     */
     protected BoxBrowseFragment getTopBrowseFragment() {
         FragmentManager fragmentManager = getSupportFragmentManager();
         Fragment frag = fragmentManager.findFragmentById(R.id.box_browsesdk_fragment_container);
         return frag instanceof BoxBrowseFragment ? (BoxBrowseFragment) frag : null;
     }
 
+    /**
+     * Handle box folder clicked.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseActivity.java]: Callback for when user clicks on a BoxFolder

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseActivity.java [hunk 2]
```diff
@@ -198,6 +218,11 @@ public void run() {
     }
 
 
+    /**
+     * Sets title.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseActivity.java]: Sets toolbar title.

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseActivity.java [hunk 3]
```diff
@@ -408,6 +436,8 @@ public IntentBuilder(final Context context, final BoxSession session) {
         }
 
         /**
+         * Sets starting folder.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseActivity.java]: Sets the folder to be browsed.

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseActivity.java [hunk 4]
```diff
@@ -419,6 +449,8 @@ public R setStartingFolder(final BoxFolder folder) {
         }
 
         /**
+         * Sets should search all.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseActivity.java]: Sets whether all folders should be searched, or only the current folder

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseFileActivity.java [hunk 5]
```diff
@@ -96,14 +98,32 @@ public static Intent getLaunchIntent(Context context, final BoxFolder folder, fi
...
+         *
+         * @param context the context
+         * @param session the session
+         */
         protected IntentBuilder(final Context context, final BoxSession session){
             super(context, session);
         }
 
+        /**
+         * Set allowed extensions intent builder.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseFileActivity.java]: Sets a list of allowed extensions to filter files by.

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseFileActivity.java [hunk 6]
```diff
@@ -7,18 +7,19 @@
...
 import com.box.androidsdk.content.models.BoxItem;
 import com.box.androidsdk.content.models.BoxSession;
-import com.box.androidsdk.content.requests.BoxRequestsSearch;
 import com.box.androidsdk.content.utils.SdkUtils;
 import com.eclipsesource.json.JsonObject;
 
 import java.util.ArrayList;
 
+/**
+ * The type Box browse file activity.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseFileActivity.java]: This activity is launched to navigate to a BoxFile.

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseFolderActivity.java [hunk 7]
```diff
@@ -29,6 +29,9 @@
 import java.net.HttpURLConnection;
 
 
+/**
+ * The type Box browse folder activity.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseFolderActivity.java]: This activity is launched to navigate a BoxFolder

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java [hunk 8]
```diff
@@ -36,7 +36,11 @@
 import java.util.concurrent.locks.ReadWriteLock;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 
+/**
+ * The type Box item adapter.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java]: Adapter for BoxItems used in the RecyclerView of a BrowseFragment

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java [hunk 9]
```diff
@@ -98,6 +112,12 @@ public void onBindViewHolder(BoxItemViewHolder boxItemHolder, int i) {
         boxItemHolder.bindItem(item);
     }
 
+    /**
+     * Get position map hash map.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java]: Get a HashMap representing item ids as keys, and positions as values.

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java [hunk 10]
```diff
@@ -424,18 +450,53 @@ public long getItemId(int position) {
     }
 
 
+    /**
+     * The type Box item view holder.
+     */
     public class BoxItemViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener, View.OnLongClickListener {
+        /**
+         * The M item.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java]: Please redo the variables.

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java [hunk 11]
```diff
@@ -665,10 +779,36 @@ public void onClick(View v) {
         }
     }
 
+    /**
+     * The interface On interaction listener.
+     */
     public interface OnInteractionListener {
+        /**
+         * Gets multi select handler.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java]: returns the MultiSelectHandler set on BrowseFragment.

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxCreateFolderFragment.java [hunk 12]
```diff
@@ -17,6 +17,9 @@
 import com.box.androidsdk.content.models.BoxSession;
 import com.box.androidsdk.content.utils.SdkUtils;
 
+/**
+ * The type Box create folder fragment.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxCreateFolderFragment.java]: A dialog fragment shown when ..

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxFilterSearchResultsFragment.java [hunk 13]
```diff
@@ -21,6 +21,9 @@
 
 import java.util.HashMap;
 
+/**
+ * The type Box filter search results fragment.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxFilterSearchResultsFragment.java]: This Fragment allows users to set filters on search parameters.

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/uidata/LoaderDrawable.java [hunk 14]
```diff
@@ -65,18 +73,34 @@ public boolean matchesRequest(final BoxRequest request){
         return false;
     }
 
+    /**
+     * The type Thumbnail task.
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/uidata/LoaderDrawable.java]: This class is a BoxFutureTask used for downloading thumbnails.

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/uidata/LoaderDrawable.java [hunk 15]
```diff
@@ -128,8 +170,19 @@ public static ThumbnailTask create(final BoxRequestsFile.DownloadThumbnail reque
     }
 
 
+    /**
+     * The interface Image ready listener.
+     */
```
Comment (seema-at-box) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/uidata/LoaderDrawable.java]: Callback interface for signifying that the thumbnail has been loaded into the imageview.

**Comment by seema-at-box** [comment]
+1

**Pull request merged**